content/en/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -103,11 +107,14 @@ $ curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
 Using `jsonpath`:
 
 ```shell
-$ APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
-$ SECRET_NAME=$(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}')
-$ TOKEN=$(kubectl get secret $SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode)
+APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+SECRET_NAME=$(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}')
+TOKEN=$(kubectl get secret $SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode)
+
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
 
```

Comment:
please add The output is similar to this: as per the guidelines before any output